src/api/app/models/staging_workflow.rb

```diff
@@ -1,4 +1,14 @@
 class StagingWorkflow < ApplicationRecord
   belongs_to :project, inverse_of: :staging
-  has_many :staging_projects, class_name: 'Project', inverse_of: :staging_workflow, dependent: :nullify
+  has_many :staging_projects, class_name: 'Project', inverse_of: :staging_workflow, dependent: :nullify, autosave: true
+
+  after_initialize :init_staging_projects
+
+  def init_staging_projects
+    if new_record?
+      ['A', 'B'].each do |letter|
```

Comment:
What happened with my comment about the constant? Although it was just a nitpick...

Reply:
I answered. Are we going to use the constant in somewhere else? If not I wont add it, it doesn't help in any way.